datavault-worker/src/main/java/org/datavaultplatform/worker/tasks/Audit.java

```diff
@@ -90,34 +94,21 @@ private void multipleCopies(Context context, Device archiveFs) throws Exception
 
         List<String> locations = archiveFs.getLocations();
         Iterator<String> locationsIt = locations.iterator();
-        LOCATION:
+
         while (locationsIt.hasNext()) {
             String location = locationsIt.next();
             logger.info("Current " + location);
             try {
                 logger.info("Attempting audit on archive from " + location);
                 this.doAudit(context, archiveFs, false, location);
                 logger.info("Completed audit on archive from " + location);
```

Comment:
I've copied the code from retrieve without thinking and this `break` was the reason why both locations were not checked as it would leave the loop once it's done with first location (which makes sense in a retrieve but not in the audit).